crates/release-automation/src/lib/crate_selection/mod.rs

```diff
@@ -80,6 +81,155 @@ impl<'a> Crate<'a> {
         self.package.manifest_path()
     }
 
+    /// Sets the new version for the given crate, updates all workspace dependants,
+    /// and returns a refrence to them for post-processing.
+    pub(crate) fn set_version(
+        &'a self,
+        dry_run: bool,
+        release_version: &semver::Version,
+    ) -> Fallible<Vec<&'a Crate<'a>>> {
+        debug!(
+            "setting version to {} in manifest at {:?}",
+            release_version,
+            self.manifest_path(),
+        );
+
+        let release_version_str = release_version.to_string();
```

Comment:
```suggestion
        let release_version_str = release_version.to_string().as_str();
```

You might as well get a `str` directly here.